<launch>
    <param name="acc_noise"       type="double" value="1e-2" />
    <param name="gyro_noise"      type="double" value="1e-4" />
    <param name="acc_bias_noise"  type="double" value="1e-6" />
    <param name="gyro_bias_noise" type="double" value="1e-8" />

    <param name="poi_gps2imu_x"       type="double" value="0.0" />
    <param name="poi_gps2imu_y"       type="double" value="0.0" />
    <param name="poi_gps2imu_z"       type="double" value="0.0" />

    <param name="poi_lidar2imu_x"       type="double" value="0.0" />
    <param name="poi_lidar2imu_y"       type="double" value="0.0" />
    <param name="poi_lidar2imu_z"       type="double" value="0.0" />

    <param name="box_size_map"       type="double" value="200" />
    <param name="voxel_size_map"     type="double" value="1.0" />
    <param name="voxel_size_frame"   type="double" value="1.0" />
    <param name="ndt_resolution"     type="double" value="2.0" />

    <param name="log_folder"      type="string" value="$(find imu_gps_localization)" />

    <param name="map_path"        type="string" value="/home/will/downloads/dataset/huashanlu/cloudMap/csf_slam/voxel10/1639710479_mapfull.pcd" />

    <node pkg="rviz" type="rviz" name="rviz" output="screen" 
      args="-d $(find imu_gps_localization)/ros_wrapper/rviz/default.rviz" required="true">
    </node>

    <!-- <node name="nmea_topic_driver" pkg="nmea_navsat_driver" type="nmea_topic_driver" output="screen" /> -->
    <node name="imu_gps_localization_node" pkg="imu_gps_localization" type="imu_gps_localization_node" output="screen" />

</launch>